packages/x-data-grid/src/models/gridRows.ts

```diff
@@ -116,9 +116,9 @@ export interface GridDataGroupNode extends GridBasicGroupNode {
 
 export interface GridDataSourceGroupNode extends GridDataGroupNode {
   /**
-   * If true, this node has children on server.
+   * Number of children this node has on the server. Returns `-1` if there are some children but the count is unknown.
```

Comment:
It's a number, not a function, right?
```suggestion
   * Number of children this node has on the server. Equals to `-1` if there are some children but the count is unknown.
```